langstream-cli/src/main/java/ai/langstream/cli/commands/docker/LocalRunApplicationCmd.java

```diff
@@ -45,6 +51,8 @@ public class LocalRunApplicationCmd extends BaseDockerCmd {
 
     protected static final String LOCAL_DOCKER_RUN_PROFILE = "local-docker-run";
 
+    private static final Set<Path> temporaryFiles = new HashSet<>();
```

Comment:
use a Concurrent set, this structure is accessed from multiple thread

Reply:
done